config/software/cryptography.rb

```diff
@@ -0,0 +1,11 @@
+name "cryptography"
+default_version "1.6"
+
+dependency "python"
+dependency "pip"
+dependency "openssl-windows"
```

Comment:
forgot about this when I asked to add `cryptography` as a dep of both `unix` and `windows` platforms: `openssl-windows` should only be specified as a dep on windows (and for unix we should specify `openssl` I guess?)